src/pages/kan-du-finne-jobb/KanDuFinneJobb.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+
+import AlternativGruppe from "../../components/alternativ-gruppe/alternativ-gruppe";
+import { Row, Column, Container } from 'nav-frontend-grid'
+
+interface KanDuFinneJobbSpmProps {
+    nextPage?: () => void,
+    endreAlternativ: (arg: string) => void,
+    valgtAlternativ: string
+}
+
+class KanDuFinneJobbSpm extends React.Component<KanDuFinneJobbSpmProps> {
+
+    static Id = "finne-jobb";
```

Comment:
?

Reply:
Samme som over, men for det andre skjermbildet.